### PR TITLE
chore: add docker-compose file for ARM64 machines

### PR DESCRIPTION
**Pull request by aqrln**

### CODE REVIEW

**Comment by millsp** [comment]
Are postgres and mongo going to work?

Not sure what this is worth or if it works:
https://hub.docker.com/r/arm64v8/mongo/
https://hub.docker.com/r/arm64v8/postgres/

**Comment by aqrln** [comment]
@millsp everything else besides MySQL and MS SQL supports ARM out-of the box with official images

File: docker/README.md [hunk 1]
```diff
@@ -16,9 +16,14 @@ If they are already used make sure to change the ports like this
...
 ports:
-  - '3307:3306 # only change the first number
+  - '3307:3306' # only change the first number
 ```
 
+## Apple M1 and other ARM64 CPUs
+
+Docker images for MySQL and Microsoft SQL Server only support x86_64 CPUs.
+Copy or link `docker-compose.arm64.yml` to `docker-compose.override.yml` to use ARMv8-compatible drop-in replacements (MariaDB and Azure SQL Edge) instead.
+
```
Comment (Jolg42) [inline docker/README.md:26]:
```suggestion
By default, Compose reads two files, a `docker-compose.yml` and an optional `docker-compose.override.yml` file. By convention, the `docker-compose.yml` contains your base configuration. The override file, as its name implies, can contain configuration overrides for existing services or entirely new services.
```

Copy pasted from docs, didn't know this

**Review by Jolg42 — APPROVED** [review]
Nice! Thanks!

**Pull request merged**